src/modules/shared/const/query-params.ts

```diff
@@ -0,0 +1,30 @@
+import { decodeString, encodeString, QueryParamConfig } from 'use-query-params';
+
+const QUERY_PARAM_SORT_DIRECTIONS = ['asc', 'desc'] as const;
+type SortDirectionTuple = typeof QUERY_PARAM_SORT_DIRECTIONS;
+type SortDirection = SortDirectionTuple[number];
+
+export function isSortDirection(value: string): value is SortDirection {
+	return QUERY_PARAM_SORT_DIRECTIONS.includes(value as SortDirection);
+}
+
+/** Uses a comma to delimit entries. e.g. ['a', 'b'] => qp?=a,b */
+export const SortDirectionParam: QueryParamConfig<string, string | undefined> = {
```

Comment:
Heeft de `useQueryParams` hier al geen standaard Param config voor `DelimitedArrayParam` en `DelimitedNumericArrayParam` ?

Reply:
Comment is unrelated, overblijfsel van code uit docs. Deze QueryParamConfig zorgt ervoor dat enkel 'asc' & 'desc' geparsed worden.